package.json

```diff
@@ -1,8 +1,27 @@
 {
   "private": true,
   "name": "msw-storybook-addon",
-  "workspaces": ["packages/*"],
+  "workspaces": [
+    "packages/*"
+  ],
   "scripts": {
     "start": "yarn workspace docs sb:start"
+  },
+  "keywords": [
+    "storybook-addon",
+    "msw",
+    "mocking",
+    "api",
+    "graphql",
+    "network",
+    "data-state"
+  ],
+  "description": "Mock API requests in Storybook with MSW",
+  "storybook": {
+    "icon": "https://msw-sb.netlify.app/logo.png",
+    "displayName": "Mock Service Worker",
+    "unsupportedFrameworks": [
+      "react-native"
```

Comment:
Hey, @coderkevin. Can you please share some more insights as to why `react-native` is listed as unsupported?

Reply:
I had added this storybook metadata to the right package.json. I think Kevin put it here on that basis. I listed React Native as unsupported because I assumed that since it runs on native platform where service workers are not supported, our addon won't work. Although, I don't have any experience is RN so I could be wrong.

Reply:
@itaditya we do support React Native through Node.js request interception. We may have some bundling issues at the moment, but the support is there, so I think we can remove this.

Reply:
Adding to this, the latest version of this addon will fallback to the server implementation of msw if the environment is not a browser, so I think we're safe! Would be nice to give a proper test though, but I did check some React native + msw code and it seems like the addon might just do the trick already.